toolkit/themes/shared/design-system/config.js

```diff
@@ -7,6 +7,8 @@
 const StyleDictionary = require("style-dictionary");
 const { formattedVariables } = StyleDictionary.formatHelpers;
 
+const BASE_SELECTOR = ":root" + "\n" + ":host(.anonymous-content-host)" + " ";
```

Comment:
Seems like there should be a comma there 🕵️
```suggestion
const BASE_SELECTOR = ":root," + "\n" + ":host(.anonymous-content-host)" + " ";
```

Maybe time to use a template string?

Reply:
heh whoops 😅